docs/testing.md

```diff
@@ -2,17 +2,49 @@
 
 ## Description
 
-This guide provides information on testing -------  .
+This guide provides information on testing FHIR Loader capabilities
 
 ## What to focus on
 
-While the team remains diligently focused on security and performance, as we introduce new features there may be times where changes may impact functionality. 
+While the team remains diligently focused on security and [performance](./performance.md), as we introduce new features there may be times where changes may impact functionality.  The tests below are focused on functionality, please see the performance link above for questions on performance.  
```

Comment:
Maybe reword this sentence to "We remain diligently focused on security and performance. As we introduce new features, there may be times where changes may impact functionality."